src/Index/Resetter.php

```diff
@@ -104,7 +104,7 @@ public function resetIndex($indexName, $populating = false, $force = false)
         }
```

Comment:
Maybe missing ` LegacyEventDispatcherProxy::decorate($dispatcher);` in constructor?

Reply:
Should be fixed, thanks for reporting forgot about it yesterday before I went home.